pkg/framework/provider/crypto/crypto.go

```diff
@@ -0,0 +1,136 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// Package crypto provides the default implementation of pkg/framework/spi/crypto.Crypto interface
+//
+// It uses github.com/tink/go crypto primitives
+package crypto
```

Comment:
Wouldn't a concrete implementation of crypto for tink be `tinkcrypto`?

Reply:
renamed package to provider/tinkcrypto instead